tools/ark-plugins/README.md

```diff
@@ -13,3 +13,45 @@ Run a Docker image build from the [Dockerfile](Dockerfile) as an init container
 ```bash
 ark plugin add {yourRepo/imageName:tag}
 ```
+
+## Add a plugin
+
+- New plugins must be added under [`internal/plugins`](internal/plugins) :
+
+```             
+  ├── internal                                                                  
+    ├── plugins
+      ├── backup    # new Backup Item Action plugins 
+      ├── restore   # new Restore Item Action plugins 
+  ```
+
+- All new plugin needs to be registered in the file `tools/ark-plugins/main.go`
+- **Backup Item Action** - performs arbitrary logic on individual items prior to storing them in the backup file.
```

Comment:
same as above, I'm not against adding subfolders but I don't see any backup actions yet

Reply:
I understand but we can't tell that **`BIA`** won"t be used. Actually part of the issue we had when the backup of lambdas is executed might solve with **`BIA`**.

This error is present after executing the restore.

```Events:
  Type     Reason                               Age   From                                Message
  ----     ------                               ----  ----                                -------
  Warning  ProvisionCallFailed                  5m    service-catalog-controller-manager  Error provisioning ServiceInstance of ClusterServiceClass (K8S: "a2257daa-0e26-4c61-a68d-8a7453c1b767" ExternalName: "redis") at ClusterServiceBroker "core-helm-broker": Status: 400; ErrorMessage: while inserting instance to storage: element already exists; Description: ; ResponseError: <nil>
  Warning  ClusterServiceBrokerReturnedFailure  5m    service-catalog-controller-manager  Error provisioning ServiceInstance of ClusterServiceClass (K8S: "a2257daa-0e26-4c61-a68d-8a7453c1b767" ExternalName: "redis") at ClusterServiceBroker "core-helm-broker": Status: 400; ErrorMessage: while inserting instance to storage: element already exists; Description: ; ResponseError: <nil>
```

Reply:
ok, the same as above